push.sh

```diff
@@ -66,9 +66,16 @@ fi
 
 indent() { sed 's/^/  /'; }
 
+declare HELM3_VERSION="helm --client --short version | grep "v3\.""
```

Comment:
This doesn’t actually run the helm command.  Need to wrap with `$()`

Reply:
Yes, you're right. Didn't check exact var declaration. Sorry. Fixed.